content/millennium/r4/scheduling/slot.md

```diff
@@ -0,0 +1,172 @@
+ `-location`      | No                         | [`reference`] | A single or comma separated list of Location references. Example: `5678`
+ `start`          | See Notes                  | [`date`]      | The Slot date-time. Example: `start=ge2016-08-24T12:00:00.000Z&start=lt2017-01-24T12:00:00.000Z`
+ [`_count`]       | No                         | [`number`]    | The maximum number of results to be returned per page.
+ [`_include`]     | No                         | [`string`]    | Other resource entries to be returned as part of the bundle. Example: `_include=Slot:schedule`
+
+Notes:
+
+* The `service-type`, `schedule.actor`, and `-location` parameters may be included only once.
+  For example, `-location=123,456` is supported but `-location=123&-location=456` is not.
+
+* `service-type` is a required search parameter when `_id` is not provided.
+
+* The `service-type` parameter may contain standard codes or proprietary codes. If multiple `service-type` parameters are used they must be either all standard codes or all proprietary codes.
+  * When using standard codes, choose codes bound to Slot.serviceType as noted in the [Terminology Bindings](#terminology-bindings) table above.
+  * When using proprietary codes, the system should be `https://fhir.cerner.com/<your EHR source id>/codeSet/<code set>` (where code set is a Millennium code set) and the code should be a Millennium code value. Example: `https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14249|24477854`
+
+* The start parameter must be provided when `service-type`, `-location`, or `actor` is provided.
+
+* When the `start` parameter is provided:
+  * both the prefixes `ge` and `lt` must be used to indicate a specific range.The date and prefix pairs must define 
+  an upper and lower bound. (e.g. &start=ge2020-04-08T18:13:55Z&start=lt2020-06-08T18:13:55Z)
+
+* The retrieved slots are sorted first by `start` date-time ascending (earliest first), followed by `service-type` and `Scheduling Location`.
+
+* The search operation will only return free slots which are available to be booked. However, booked slots may still be retrieved when using the `_id` parameter.
+
+* The `_include` parameter may be provided once with the value `Slot:schedule`. Example: `_include=Slot:schedule`
+
+* The `_include` parameter may be provided with the `_id` parameter. Example: `_id=21265426-633867-3121665-0&_include=Slot:schedule`
+
+* When `_include` is provided in a request to the closed endpoint, the OAuth2 token must include either the `user/Schedule.read` or the `patient/Schedule.read` scope in addition to a Slot scope.
+
+### Headers
+
+ <%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.devcerner.com/r4/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/Slot?service-type=https://fhir.cerner.com/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/codeSet/14249|517975329&start=ge2020-07-22T20:04:19Z&start=lt2020-11-08T18:13:55Z&schedule.actor=Practitioner/2911963
```

Comment:
Change the url from 'fhir-open.devcerner.com' to 'fhir-open.cerner.com'

Reply:
Also use sandbox tenant: https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/

Reply:
updated all URLs with sandbox url